packages/fast-tooling-react/src/navigation-menu/navigation-menu-item.props.ts

```diff
@@ -0,0 +1,44 @@
+import { ComponentStyleSheet, ManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { MenuItem } from "./navigation-menu.props";
+
+export interface NavigationMenuItemClassNameContract {
+    navigationMenuItem?: string;
+    navigationMenuItem_list?: string;
+    navigationMenuItem_list__expanded?: string;
+    navigationMenuItem_listItem?: string;
+    navigationMenuItem_listItem__active?: string;
+}
+
+export interface NavigationMenuItemState {
+    expanded: boolean;
+}
+
+export interface NavigationMenuItemUnhandledProps
+    extends React.HTMLAttributes<HTMLElement> {}
+
+export interface NavigationMenuItemHandledProps
+    extends ManagedClasses<NavigationMenuItemClassNameContract> {
+    /**
+     * The menu item
+     */
+    menuItem: MenuItem;
```

Comment:
Since this is already a menu-item it seems a bit clumsy to have a `menuItem` property - could this prop interface extend `MenuItem` instead?